plugin/src/main/scala/eu/unicredit/swagger/SwaggerCodegenPlugin.scala

```diff
@@ -121,7 +121,7 @@ object SwaggerCodegenPlugin extends AutoPlugin {
 
   override val projectSettings = {
     Seq(
-      watchSources ++= swaggerSourcesDir.value.***.get,
+      watchSources ++= swaggerSourcesDir.value.allPaths.get,
```

Comment:
this will work only with sbt `1.X` at the best of my understanding. You do need to refactor and have separate directories for code that is for `0.13` and for `1.X` or find an equivalent method that is API compatible.

Reply:
Holy macarons!